clc/modules/object-storage/src/main/java/com/eucalyptus/objectstorage/ObjectStorageGateway.java

```diff
@@ -877,14 +898,23 @@ public PostObjectResponseType postObject(PostObjectType request) throws S3Except
    */
   @Override
   public DeleteObjectResponseType deleteObject(final DeleteObjectType request) throws S3Exception {
-    ObjectEntity objectEntity;
+    ObjectEntity objectEntity = null;
+    Bucket bucket = null;
     try {
+      //LPT Bug! Throws exception on anonymous access to an object
+      //bucket = getBucketAndCheckAuthorization(request);
+      //LPT The fix:
```

Comment:
This is a temporary change to be able to test anonymous accesses until EUCA-12174 is fixed. I should have said "The workaround for branch testing" instead of "The fix". It would never be in code I pushed beyond my branch, the auth check would remain, but in a way that also allows anonymous access.